### ISSUE TITLE
Still getting errors on launch: Error loading script: tag_autocomplete_helper.py

### ISSUE THREAD

**Issue by vt-idiot** [DominikDoom/a1111-sd-webui-tagcomplete#205]
```py
*** Error loading script: tag_autocomplete_helper.py
    Traceback (most recent call last):
      File "/content/stable-diffusion-webui/modules/scripts.py", line 274, in load_scripts
        script_module = script_loading.load_module(scriptfile.path)
      File "/content/stable-diffusion-webui/modules/script_loading.py", line 10, in load_module
        module_spec.loader.exec_module(module)
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/content/stable-diffusion-webui/extensions/a1111-sd-webui-tagcomplete/scripts/tag_autocomplete_helper.py", line 134, in <module>
        model_keyword_installed = write_model_keyword_path()
      File "/content/stable-diffusion-webui/extensions/a1111-sd-webui-tagcomplete/scripts/model_keyword_support.py", line 65, in write_model_keyword_path
        mk_path.write_text("")
      File "/usr/lib/python3.10/pathlib.py", line 1154, in write_text
        with self.open(mode='w', encoding=encoding, errors=errors, newline=newline) as f:
      File "/usr/lib/python3.10/pathlib.py", line 1119, in open
        return self._accessor.open(self, mode, buffering, encoding, errors,
    FileNotFoundError: [Errno 2] No such file or directory: '/content/stable-diffusion-webui/tmp/modelKeywordPath.txt'
```

**Comment by vt-idiot** [comment]
I was able to fix it with a pretty simple `mkdir -p /content/stable-diffusion-webui/tmp/` - now I only see the expected `Tag Autocomplete: Could not locate model-keyword extension, LORA/LYCO trigger word completion will be unavailable.` while launching.

**Comment by yo1nkers** [comment]
Hi, is the current issue you're experiencing
`Tag Autocomplete: Could not locate model-keyword extension, LORA/LYCO trigger word completion will be unavailable.`
causing you any major issues? This is currently the issue I am getting, and I am looking for a fix to this.

**Comment by DominikDoom** [comment]
@yo1nkers Is this unexpected for you or preventing normal completion from working? If you just see that message but nothing else, all it means is that the keyword lists weren't found. It's just an info message so people know why the feature would be unavailable in that case.

**Comment by DominikDoom** [comment]
Exactly, if you don't want to use it you can just ignore the message. Since some seem to think of it as a critical error I will maybe reword or remove the hint in the future though.

The list of model keywords provided by that extension is pretty big and auto updated from civitai, so it was a good way to not have to support and ship my own list.
The README has a [new section](https://github.com/DominikDoom/a1111-sd-webui-tagcomplete#lora--lyco-trigger-word-completion) about it that goes into detail.

**Comment by brehuang** [comment]
Having the same issue. This seems to have coincided with the new release of automatic1111, since I didn't have this issue yesterday. I can't use Latent Couple anymore. If anyone has a solve, please share.

**Comment by DominikDoom** [comment]
@brehuang What issue exactly are you talking about? The error in the first comment here which was already fixed, or the "Could not locate model-keyword extension" info message which is harmless?
This extension is not affected by the new webui release as far as I'm aware, so anything else will be unrelated (assuming tag-autocomplete is up to date too)
